qemu/tests/image_create_with_large_size.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from avocado.core import exceptions
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    Creating image with large size.
+    1. Create a qcow2 image with large size and check the output info.
+    2. Create a qcow2 with a normal size.
+    3. Increase a large size to the qcow2 image and check the output info.
+
+    :param test: VT test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    large_img, small_img = params["images"].split()
+    root_dir = data_dir.get_data_dir()
+    large = QemuImg(params.object_params(large_img), root_dir, large_img)
+    small = QemuImg(params.object_params(small_img), root_dir, small_img)
+    large_filename = large.image_filename
+    size_increases = params["size_changes"]
+    create_err_info = "The image size is too large for file format " \
+                      "'qcow2' (try using a larger cluster size)"
+    resize_err_info = "Failed to grow the L1 table: File too large"
```

Comment:
Suggest that move the `create_err_info` and `resize_err_info` to cfg file to avoid hard code.
Thanks.

Reply:
Updated, thanks.